images/oc-build-deploy-dind/build-deploy-docker-compose.sh

```diff
@@ -961,6 +961,11 @@ elif [ "$TYPE" == "pullrequest" ] || [ "$TYPE" == "branch" ]; then
     parallel --retries 4 < /oc-build-deploy/lagoon/push
   fi
 
+  # If we have Images to Push to the the Internal Harbor, let's do so
+  if [ -f /oc-build-deploy/lagoon/push_harbor ]; then
```

Comment:
can you run them also with `--retries 4`? just to be sure they actually end up in harbor :)

Reply:
Done! :)